docs/odh/oci-model-storage.md

```diff
@@ -0,0 +1,208 @@
+```
+
+The template that is applicable for KServe and holds the OpenVINO configuration
+is the one named as `kserve-ovms`. To create an instance of it, run the
+following command:
+```shell
+oc process -n opendatahub -o yaml kserve-ovms | oc apply -f -
+```
+
+You can verify that the `ServingRuntime` has been created successfully with the
+following command:
+```shell
+oc get servingruntimes
+
+NAME          DISABLED   MODELTYPE     CONTAINERS         AGE
+kserve-ovms              openvino_ir   kserve-container   1m
+```
+
+Notice that the ServingRuntime has been created with `kserve-ovms` name.
+
+Now that the `ServingRuntime` is configured, a model stored in an OCI image can
+be deployed by creating an `InferenceService` resource:
+```yaml
+apiVersion: serving.kserve.io/v1beta1
+kind: InferenceService
+metadata:
+  name: sample-isvc-using-oci
+spec:
+  predictor:
+    model:
+      runtime: kserve-ovms # This is the name of the ServingRuntime resource
+      modelFormat:
+        name: onnx
+      storageUri: oci://quay.io/<user_name>/<repository_name>:<tag_name>
+```
+
+> [!IMPORTANT]
+> The resulting `ServingRuntime` and `InferenceService` configurations won't set
+> any resource limits.
+
+Once the `InferenceService` resource is created, KServe will deploy the model
+stored in the OCI image referred by the `storageUri` field. Check the status
+of the deployment with the following command:
+```shell
+oc get inferenceservice
+
+NAME                    URL                                                       READY   PREV   LATEST   PREVROLLEDOUTREVISION   LATESTREADYREVISION                     AGE
+sample-isvc-using-oci   https://sample-isvc-using-oci-oci-model-example.example   True           100                              sample-isvc-using-oci-predictor-00001   1m
+```
+
+> [!IMPORTANT]
+> Remember that, by default, models are exposed outside the cluster and not
+> protected with authorization. Read the [authorization guide](authorization.md#deploying-a-protected-inferenceservice)
+> and the [private services guide (TODO)](#TODO) to learn how to privately deploy
+> models and how to protect them with authorization.
+
+## Deploying a model stored in an OCI image from a private repository
+
+To deploy a model stored in a private OCI repository you need to configure an
+image pull secret. For detailed documentation, please [consult the OpenShift
+documentation for image pull secrets](https://docs.openshift.com/container-platform/latest/openshift_images/managing_images/using-image-pull-secrets.html).
+
+When using namespaced pull secrets you can create a pull secret using the following
+command template:
+
+```shell
+oc create secret docker-registry <pull-secret-name> \
+  --docker-server=<registry-server> \
+  --docker-username=<username> \
+  --docker-password=<password>
+```
+
+Once the pull secret is created, you can follow the steps from the previous
```

Comment:
what is the previous section?

Reply:
i.e. the previous heading _Deploying a model stored in an OCI image in a public repository_

Reply:
I guess I didn't find it

Reply:
oops.